Dockerfile

```diff
@@ -16,7 +16,12 @@ RUN /bin/bash -c "source activate urlchecker && \
     which python && \
     which pip && \
     pip install --upgrade certifi && \
-    pip install ."
+    pip install .[all]"
+# Download chrome driver for selenium
+RUN wget https://chromedriver.storage.googleapis.com/104.0.5112.29/chromedriver_linux64.zip && \
+    rm -rf chromedriver && \
```

Comment:
I really dislike `rm`s; they often mess things up for me so I try to avoid them. Do we really need this here?

Reply:
Yes, if the user builds the container locally it will add the local version to the same spot and the extraction will fail. It confused me a few minutes until I realized what was happening. There isn't any risk here because it's inside the container, and it's targeting that one file. The `-rf`is needed so it doesn't error if the driver does not exist.